docs/specification.md

```diff
@@ -92,6 +92,8 @@ Required `Safelist` entrypoints:
 * `assertReceivers(list address)`
```

Comment:
Note that apparently `assertReceiver` was removed, so we should remove it as well and use only `assertReceivers`.